mlflow/tracking/fluent.py

```diff
@@ -3472,9 +3472,31 @@ def _get_active_model_id_global() -> Optional[str]:
     _logger.debug("No active model ID found in any thread.")
 
 
-def _reset_active_model_context() -> None:
+def unset_active_model() -> None:
     """
-    Should be called only for testing purposes.
+    Unset the active model. This will unset the active model previously set by
+    :py:func:`mlflow.set_active_model` from current thread. To temporarily switch
+    the active model, use ``with mlflow.set_active_model(...)`` instead.
+
+    .. code-block:: python
+        :test:
+        :caption: Example
+
+        import mlflow
+
+        # Set the active model by name
+        mlflow.set_active_model(name="my_model")
+
+        # Unset the active model
+        mlflow.unset_active_model()
+        # Check that the active model is unset
+        assert mlflow.get_active_model_id() is None
```

Comment:
Can we also a context manager `set_active_model` in the example and recommend it?

```python
with set_active_model():
   ...
```